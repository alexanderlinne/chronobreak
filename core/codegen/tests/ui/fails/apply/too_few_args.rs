#[macro_use]
extern crate chronobreak_derive;

pub use chronobreak::{clock, mock};

fn main() {
    apply!(0);
}
