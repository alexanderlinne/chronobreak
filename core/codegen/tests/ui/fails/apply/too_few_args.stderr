error: apply! expectes 2 or 3 arguments
 --> $DIR/too_few_args.rs:7:12
  |
7 |     apply!(0);
  |            ^
